File: Curious-Freaks/Beginner/Armstrong.java
//To find armstrong number
class Solution {
    static boolean armstrongNumber(int n) {
        int sum=0,temp=n,digit=(Integer.toString(n)).length();
        while(temp>0)
        {
            sum+=(int)(Math.pow(temp%10,digit));
            temp/=10;
        }
        if(n==sum)
        return true;
        else
        return false;
        
    }
}

File: Striver's/Basics/Maths/armstrong.py
#Problem Statement:Given an integer N, return true it is an Armstrong number otherwise return false.

# Complexity Time - O(logN) , Space - O(1)

import math
num=8208
temp=num
digits=int(math.log10(num)+1)
su=0
while temp>0:
    digit=temp%10
    su=su+int(math.pow(digit,digits))
    temp=temp//10
print("Armstrong number" if num==su else "Not Armstrong")